Cargo.toml

```diff
@@ -78,18 +78,16 @@ deflate-flate2-zlib = ["deflate-flate2", "flate2/zlib"]
 deflate-zopfli = ["dep:zopfli", "_deflate-any"]
 jiff-02 = ["dep:jiff"]
 nt-time = ["dep:nt-time"]
-lzma = ["dep:liblzma"]
-lzma-static = ["lzma", "liblzma/static"]
```

Comment:
Keep `lzma-static` and `xz-static` for backward compatibility, even if they have no effects except enabling `lzma` and `xz`.

Reply:
I will add them back.